oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -0,0 +1,428 @@
+Learning, it is essential to have a well defined plan for maintaining
+the platform.
+
+In order to ensure that the project is sustainable for the long term,
+maintenance must be shared by the entire community.  The project must
+be robust enough that it can tolerate firms and contributors leaving
+the ecosystem.
+
+As a project we must acknowledge that maintenance is hard work, but
+work that must be done for the platform to be fit for use.  The best
+way of ensuring this is to have a named person or persons who are
+responsible for getting the work done, a measurable definition of
+"well-maintained," and feedback looks to ensure accountability and
+continuous improvement of our processes.
+
+Specification
+=============
+
+Definitions
+-----------
+
+The term **repository** will mean a repository on the GitHub social
+coding platform.
+
+The term **component** will typically mean the unit of software housed
+in a single GitHub repository.  Exceptions will be made in the case of
+edx-platform where there may be multiple distinguishable components in
+that single repository.  Components will typically, but not always, be
+hosted at the `openedx GitHub organization`_. Components will be "part
+of" or "critical to" the open-source platform.
+
+The term **maintainer** will mean a named person or persons
+responsible for the health of the component and the component's community.
+The remainder of this specification will be focused on the details of
+the role.
+
+The term **core contributor** will mean a named person participating
+in the `Core Contributor Program`_ as a coding core contributor and
+meeting the requirements of that program.
+
+The term **maintainer scrum-of-scrums** will mean the group of all
+maintainers who are responsible for flagging issues to project
+leadership and proposing process improvements for component
+maintenance.
+
+The term **Technical Oversight Committee** (TOC) will mean the group
+responsible for the overall oversight and stewardship of the Open edX
+project. The TOC's function with respect to the project is to be the
+arbiter of last resort for governance matters. The TOC itself was established
+and is governed by a formal
+charter.  You can read the full document `here
+<https://openedx.org/wp-content/uploads/2021/12/TCRIL-Technical-Oversight-Committee-Charter-rev.11-16.21.pdf>`_.
+
+Historical Context
+------------------
+Historically, the Open edX project has not had a well-defined role
+of maintainer.  At times, community members stepped into this role
+informally.  The role was also part of a broader "owner" role at edX.
+Owners were also responsible for the health of the component they
+owned both in the repository and in production; owners answered the
+pager when something went wrong.
+
+What Maintainers Do
+-------------------
+The role of the repository or project maintainer is both a community
+focused and a technical role [#role]_.  Maintainers will be the public face
```

Comment:
To me this feels like the role that the Core Contributor - Project Manager is meant to address. I know Jorge is working with a goal of eventually becoming a CC in this role. And again to my earlier point - I don't love the idea of a club that has a high enough bar that people in certain roles cannot hope to achieve the required monthly status - that makes some roles feel lesser than others. A tiered system also has that feeling. I would like to discuss how we can be inclusive and friendly, while asking for and celebrating higher commitments of some people, groups, or ecosystem partners.

Reply:
Hi @antoviaque @adamstankiewicz - is this thread a blocking concern for merging this OEP to provisional status?

Personally I am ok with the current wording. "Technical" does not need to mean you write code. It does mean you understand the technical ins & outs of GitHub, for example. It may mean you understand how to debug RST or work in other domain-specific areas. What do you think? I would like either a "this is ok as-is and we can iterate as needed", or a proposed wording change. Thanks!

Reply:
@sarina Since it's already a large OEP with a lot of changes, I'm fine with merging as-is for now. After all, that's also how we started it for core committers, by focusing on the technical/dev roles. Once we have seen a bit how maintainers work out on a smaller set of duties, we can always extend it then to other roles if it looks pertinent.